src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceTests.cs

```diff
@@ -444,7 +445,7 @@ public class Generator : ISourceGenerator
                 var generatorPath = Path.Combine(directory.Path, $"generator_{targetFramework}.dll");
 
                 var compilation = CSharpCompilation.Create($"generator_{targetFramework}",
-                                                           new[] { CSharpSyntaxTree.ParseText(generatorSource) },
+                                                           new[] { CSharpTestSource.Parse(generatorSource) },
```

Comment:
Given that you disabled the banned api analyzer in test projects, is there still a need for this change?

Reply:
And everywhere else in tests that needed to be changed for this.

Reply:
Technically this change is not needed but it is still preferable to use the common test helper even if the banned API usage is not reported.

Reply:
I guess I'm not a fan of this either, it's really bloating the diff size generally, but now means a bunch of tests are having to call extra test helpers for reasons that aren't clear here...